astropy/io/fits/util.py

```diff
@@ -38,10 +38,10 @@ class StringIO(object):
 from ...utils.compat import suppress
 from ...utils.exceptions import AstropyUserWarning
 
-if six.PY3:
-    cmp = lambda a, b: (a > b) - (a < b)
-elif six.PY2:
+if six.PY2:
```

Comment:
This will almost certainly conflict with #4985. Just in case the other PR is merged first.